lib/web_ui/dev/test_platform.dart

```diff
@@ -178,8 +178,15 @@ To automatically create this file call matchGoldenFile('$filename', write: true)
         },
       };
     }
-    // To tweak DPI we need to send an additional Emulation.setDeviceMetricsOverride. See:
+
+    // Setting hardware-independent screen parameters:
     // https://chromedevtools.github.io/devtools-protocol/tot/Emulation
+    await wipConnection.sendCommand('Emulation.setDeviceMetricsOverride', {
+      'width': 1024,
```

Comment:
There are (poorly named) constants for this, used also in the headless chrome command line parameters:

`_kMaxScreenshotWidth` and `_kMaxScreenshotHeight`

Reply:
👍